tasks/windows.task/second-stage.ps1.erb

```diff
@@ -16,10 +17,10 @@ $unattended = "${env:SYSTEMDRIVE}\unattended.xml"
 (new-object System.Net.WebClient).DownloadFile($unattended_src, $unattended)
 
 
-write-host "mapping SMB share \\${server}\${share} for installer access"
+write-host "mapping SMB share ${drive_path} for installer access"
 # As far as I can tell, if we don't persist, we don't connect the SMB share,
 # and that means we can't actually run stuff off it.
-new-psdrive -name 'i' -psprovider filesystem -root "\\${server}\${share}" -persist
+new-psdrive -name 'i' -psprovider filesystem -root "$($drive_path)" -persist
```

Comment:
It was intentional. I saw no reason to have the newly formed I: drive contain access to other repos. Rather than having all paths access I:/$repo/$file_path, it'll just be I:/$file_path, which I think is simpler.